class.jetpack-sync.php

```diff
@@ -1101,4 +1103,19 @@ static function get_constant( $constant ) {
 
 		return null;
 	}
+
+	function log_automatic_updates( $results ) {
+		if ( empty( $results ) ) {
+			return;
+		}
+		$log = Jetpack_Options::get_option( 'updates_log', array() );
+		// Append our event to the log
+		$log[] = array(
+			'time'    => time(),
+			'user_id' => get_current_user_id(),
+			'results'    => $results,
+		);
+		Jetpack_Options::update_option( 'updates_log', $log );
```

Comment:
We've already been creating a log here:
https://github.com/Automattic/jetpack/blob/master/class.jetpack-autoupdate.php#L111

Perhaps the endpoint could be used to grab that? Or at the very least we should stop the old log in favor of the new, so we are not saving things twice.